static/js/tools/map/fetcher/border_geojson.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch Geo Json data for the border of the containing place.
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import { Dispatch, useContext, useEffect } from "react";
+
+import { GeoJsonData } from "../../../chart/types";
+import { ChartDataType, ChartStoreAction } from "../chart_store";
+import { Context } from "../context";
+
+// Use non-DP geojsons for drawing borders to avoid mismatched edges between
+// the border and enclosed places
+const BORDER_GEOJSON_PROPERTY = "geoJsonCoordinates";
+
+export function useFetchBorderGeoJson(
```

Comment:
nit: add comment docs to exported functions

Reply:
Added a comment docv